packages/kn-plugin-workflow/pkg/command/deploy.go

```diff
@@ -0,0 +1,154 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package command
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/kiegroup/kie-tools/kn-plugin-workflow/pkg/common"
+	"github.com/ory/viper"
+	"github.com/spf13/cobra"
+)
+
+func NewDeployCommand() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   "deploy",
+		Short: "Deploy a Quarkus workflow project",
+		Long: `Deploy a Quarkus workflow project
+
+	This command deploys the workflow project in the current directory. 
+	By default this commands uses the ./target/kubernetes folder to find
+	the deployment files generated in the build process, so the build step
+	is required before using the deploy command.
+
+	Before you use the deploy command, ensure that your cluster have 
+	access to the build output image.
+		`,
+		Example: `
+	# Deploy the workflow from the current directory's project. 
+	# Deploy as Knative service.
+	{{.Name}} deploy
+	
+	# Specify the path of the directory containing the "knative.yml" 
+	{{.Name}} deploy --path ./kubernetes
+		`,
+		SuggestFor: []string{"delpoy", "deplyo"},
+		PreRunE:    common.BindEnv("path"),
+	}
+
+	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		return runDeploy(cmd, args)
+	}
+
+	cmd.Flags().StringP("path", "p", "./target/kubernetes", fmt.Sprintf("%s path to knative deploy files", cmd.Name()))
+
+	cmd.SetHelpFunc(common.DefaultTemplatedHelp)
+
+	return cmd
+}
+
+func runDeploy(cmd *cobra.Command, args []string) error {
+	start := time.Now()
+
+	cfg, err := runDeployConfig(cmd)
+	if err != nil {
+		return fmt.Errorf("initializing deploy config: %w", err)
+	}
+
+	if _, err := exec.LookPath("kubectl"); err != nil {
+		fmt.Println("ERROR: kubectl is required for deploy")
+		fmt.Println("Download from https://kubectl.docs.kubernetes.io/installation/kubectl/")
+		os.Exit(1)
+	}
+
+	createService := exec.Command("kubectl", "apply", "-f", fmt.Sprintf("%s/knative.yml", cfg.Path))
+	if err := common.RunCommand(
+		createService,
+		cfg.Verbose,
+		"deploy",
+		getDeployFriendlyMessages(),
+	); err != nil {
+		fmt.Println("Check the full logs with the -v | --verbose option")
+		return err
+	}
+	fmt.Println("✅ Knative service sucessufully created")
+
+	// Check if kogito.yml file exists
+	if exists, err := checkIfKogitoFileExists(cfg); exists && err == nil {
+		deploy := exec.Command("kubectl", "apply", "-f", fmt.Sprintf("%s/kogito.yml", cfg.Path))
```

Comment:
```suggestion
		deploy := exec.Command("kubectl", "apply", "-f", fmt.Sprintf("%s/kogito.json", cfg.Path))
```

I think it's `kogito.json`. Can you confirm, please?